client/src/Pages/AuthPage/Loginpage.tsx

```diff
@@ -0,0 +1,52 @@
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { useAuth } from "../../context/authcontext";
+import "./AuthPage.css";
+const LoginPage = () => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const { loginHandler } = useAuth();
+
+  const guestemail = "aman@gmail.com";
+  const guestpassword = "123456";
```

Comment:
Can store these details in a single object called as guestCredentials={
guestemail : "aman@gmail.com",
guestpassword : "123456"
}
That will help reduce the work of creating multiple states. Would advise in case of fields that take email and password from the inputs.

Reply:
i havent created a state , i created two seperated variables since my loginhandler accepts two args and not a object

Reply:
Understood the purpose.